app/index.html

```diff
@@ -10,6 +10,7 @@
     <meta property="og:image" content="https://mhacks.org/logo.png"/>
 
     <link href="https://fonts.googleapis.com/css?family=Montserrat:400,500" rel="stylesheet">
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
```

Comment:
Does the react-fontawesome package require this to be imported or does it also come included with the module